flow/envs/base_env.py

```diff
@@ -507,7 +437,7 @@ def reset(self):
                 self.traci_connection.vehicle.remove(veh_id)
                 self.traci_connection.vehicle.unsubscribe(veh_id)
             except (FatalTraCIError, TraCIException):
-                print("Error during start: {}".format(traceback.format_exc()))
```

Comment:
Perhaps throw a warning before passing?

Reply:
there was a little bug. fixed it and re-introduced the error message